src/java/picard/illumina/CollectIlluminaLaneMetrics.java

```diff
@@ -49,13 +49,29 @@
  */
 
 @CommandLineProgramProperties(
-        usage = CollectIlluminaLaneMetrics.USAGE,
-        usageShort = CollectIlluminaLaneMetrics.USAGE,
+        usage = CollectIlluminaLaneMetrics.USAGE_SUMMARY + CollectIlluminaLaneMetrics.USAGE_DETAILS,
+        usageShort = CollectIlluminaLaneMetrics.USAGE_SUMMARY,
         programGroup = Illumina.class
 )
 public class CollectIlluminaLaneMetrics extends CommandLineProgram {
-    static final String USAGE = "Collects Illumina lane metrics for the given basecalling analysis directory";
-
+    static final String USAGE_SUMMARY = "Collects Illumina lane metrics for the given BaseCalling analysis directory.  ";
+    static final String USAGE_DETAILS = "This tool produces quality control metrics on cluster density for each lane of an Illumina flowcell." +
+            "  This tool works via distributing Illumina TileMetrics data into directories containing lane- and phasing-level metrics.  " +
```

Comment:
"This tool works via distributing" -> grammar is a tad strained, please rephrase